serverless.yml

```diff
@@ -55,6 +55,14 @@ provider:
 #    - exclude-me-dir/**
 
 functions:
+  master:
+    handler: mapreduce/master.master
+    events:
+      - stream:
```

Comment:
@jdd1260 here is the config for where the lambda gets triggered by the dynamodb event